x-pack/plugins/fleet/common/types/models/data_stream.ts

```diff
@@ -24,3 +24,5 @@ export interface DataStream {
     serviceName: string;
   } | null;
 }
+
+export type PackageDataStreamTypes = 'logs' | 'metrics' | 'traces' | 'synthetics' | 'profiling';
```

Comment:
Possibly not the best naming, or location? But when looking at other Fleet code I could only really find the `DATA_STREAM_INDEX_PATTERN` constant, but not a type I could reuse.